user.go

```diff
@@ -195,35 +194,6 @@ func NewUserClaimsFromMap(m map[string]interface{}) (*UserClaims, error) {
 		}
 	}
 
-	if _, exists := m["app_metadata"]; exists {
```

Comment:
@njones , please sync with the latest commit. This one is my "app_metadata" change.